packages/modular-scripts/src/esbuild-scripts/start/index.ts

```diff
@@ -74,6 +90,8 @@ class DevServer {
     this.ws = ws(this.express);
 
     this.express.use(this.handleStaticAsset);
+    this.isApp || // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
nit: this should be an if statement and would remove the eslint issues.

Reply:
It actually wouldn't - the problem is that `handleStaticAsset` returns a promise where a void function is expected, from the pov of the compiler.